lib/src/iterable_extensions.dart

```diff
@@ -793,3 +826,66 @@ extension ComparatorExtension<T> on Comparator<T> {
         return result;
       };
 }
+
+/// Extensions on sets.
```

Comment:
[nit] This comment can be omitted unless we have something more interesting to add.